frontend/src/pages/Domains/index.tsx

```diff
@@ -27,61 +44,69 @@ const useStyles = makeStyles(() =>
 const Domains: React.FC = () => {
   const classes = useStyles();
   return (
-    <div className={classes.root}>
-      <NavBar />
-
-      <Container>
+    <div>
+      <Nav2 />
+      <div className={classes.root}>
+        <Typography className={classes.heading} align="center">
+          We are everywhere
+        </Typography>
+        <Typography className={classes.subHeading} align="center">
+          {"<p>From web to app, AI to ML, Blockchain to DevOps</p>"}
```

Comment:
Why is this wrapped within `{}`?

Reply:
We want to display `<p>From web to app, AI to ML, Blockchain to DevOps</p>`, so I used `{}` to treat `<p></p>` as a string instead of HTML tag.

Reply:
Got it.